src/core/dbus-execute.c

```diff
@@ -3338,25 +3338,23 @@ int bus_exec_context_set_transient_property(
                 if (r < 0)
                         return r;
 
-                if (!isempty(s)) {
-                        if (!path_is_absolute(s))
-                                return sd_bus_error_setf(error, SD_BUS_ERROR_INVALID_ARGS, "Path %s is not absolute", s);
-                        if (!path_is_normalized(s))
-                                return sd_bus_error_setf(error, SD_BUS_ERROR_INVALID_ARGS, "Path %s is not normalized", s);
-                }
+                if (!path_is_absolute(s))
```

Comment:
Rather than always requesting a valid path, maybe better to clear assignment when an empty string is passed? Not all bus properties do so, but some do. E.g. `RestrictFileSystems=` clears assignment when an empty strv is passed.

Reply:
What would stdin mode become then? We cannot sensibly perform reset...

Reply:
OK. Even conf parsers for StandardInput=/Output=/Error= do not support an empty string assignment. Still I think both conf parsers and here should support an empty string, but let's do that later.